doc/user/content/concepts/reaction-time.md

```diff
@@ -0,0 +1,95 @@
+
+Together, these concepts form the basis for understanding how Materialize enables timely, accurate insights across operational and analytical workloads.
+
+---
+
+## Freshness
+
+**Freshness** measures the time it takes for a change in an upstream systemnto become visible in the results of a query. In other words, it captures the end-to-end latency between when data is produced and when it becomes part of the transformed, queryable state.
+
+### System Comparisons
+
+* **OLTP databases**: Freshness is effectively zero. Queries run directly against the source of truth, and changes are visible immediately.
+* **Data warehouses**: Freshness is often poor due to scheduled batch ingestion. Changes may take minutes to hours to propagate.
+* **Materialize**: Freshness is low, typically within milliseconds to a few seconds, due to continuous ingestion and incremental view maintenance.
+
+---
+
+## Query Latency
+
+**Query latency** refers to the time it takes to compute and return the result of a SQL query once the data is available in the system. It is affected by the system's execution model, indexing strategies, and the complexity of the query itself.
+
+### System Comparisons
+
+* **OLTP databases**: Optimized for transactional workloads and point lookups. Complex analytical queries involving joins, filters, and aggregations tend to exhibit poor query latency.
+* **Data warehouses**: Designed for analytical processing, and generally provide excellent query latency even for complex queries over large datasets.
+* **Materialize**: Maintains low query latency by incrementally updating and indexing the results of complex views. Queries that read from indexed views typically return results in milliseconds.
+
+---
+
+## Reaction Time
+
+**Reaction time** is defined as the sum of freshness and query latency. It captures the total time from when a data change occurs upstream to when a downstream consumer can query and act on that change.
+
+```
+reaction time = freshness + query latency
+```
+
+This is the most comprehensive measure of system responsiveness and is particularly relevant for applications that depend on timely and accurate decision-making.
+
+### System Comparisons
+
+| System         | Freshness    | Query Latency              | Reaction Time |
+| -------------- | ------------ | -------------------------- | ------------- |
+| OLTP Database  | Excellent    | Poor (for complex queries) | High          |
+| Data Warehouse | Poor (stale) | Excellent                  | High          |
+| Materialize    | Excellent    | Excellent                  | Low           |
+
+---
+
+## Illustrative Example
+
+Consider an e-commerce application that needs to monitor order fulfillment rates in real time:
+
+* In an **OLTP system**, the order and fulfillment data is current, but computing the fulfillment rate requires aggregations and joins over multiple tables, which can result in high query latency. The data is fresh, but the response is slow.
```

Comment:
Not sure .. but I wonder if making the data + computation details into 2 bullet points under each system might make it easier for users to skim and catch the differences.